itest/list_on_test.go

```diff
@@ -715,3 +711,8 @@ var allTestCases = []*lntest.TestCase{
 		TestFunc: testQuiescence,
 	},
 }
+
+func init() {
+	// Register subtests.
+	allTestCases = append(allTestCases, multiHopForceCloseTestCases...)
+}
```

Comment:
this is an intermediate version, and the final version is,
https://github.com/lightningnetwork/lnd/blob/96756b40995647427575898af6bf5d675654a780/itest/list_on_test.go#L703-L754

Since it involve a function call, think we have to do it here.

Other than that, I'm curious why avoiding `init` function?